src/components/GenericModal.vue

```diff
@@ -87,6 +91,10 @@ export default {
       type: Boolean,
       default: true,
     },
+    styleProps: {
```

Comment:
I'm not a fan of adding styles through props, because that's what CSS classes are for instead.
I know that the width is a prop at the moment but overwriting more styles through props doesn't seem an efficient idea.
I suggest you overwrite those styles applying classes using the `/deep/` element

Reply:
Thanks for the suggestion Marina 🙏 I was unaware of the `/deep/` selector. It doesn't feel right to have the style scoped and still mess with the child styling, but I agree that's a cleaner solution than passing it as a prop. Resolved [2fa4743](https://github.com/apple/swift-docc-render/pull/396/commits/2fa474366a821887f60369e3d6806267aac7abac).